salt/modules/win_network.py

```diff
@@ -344,38 +344,56 @@ def in_subnet(cidr):
     return salt.utils.network.in_subnet(cidr)
 
 
-def ip_addrs(interface=None, include_loopback=False):
+def ip_addrs(interface=None, include_loopback=False, cidr=None, type=None):
     '''
     Returns a list of IPv4 addresses assigned to the host. 127.0.0.1 is
     ignored, unless 'include_loopback=True' is indicated. If 'interface' is
     provided, then only IP addresses from that interface will be returned.
+    Providing a CIDR via 'cidr="10.0.0.0/8"' will return only the addresses
```

Comment:
I know you didn't add this function and the associated docs originally, but I'd love it if definitions for each of these kwargs could be added. There should be examples of how to do this in other modules. Then you could add a `.. versionadded:: Fluorine` tag to the new options and make it easy for people to see when these were added.